src/wfs/url.ts

```diff
@@ -57,7 +58,18 @@ export function generateGetFeatureUrl(
     newParams.STARTINDEX = startIndex.toString(10);
   }
 
-  return setQueryParams(serviceUrl, newParams);
+  let url = setQueryParams(serviceUrl, newParams);
+
+  // Don't encode +A or +D Wfs sorting param
+  if (Array.isArray(sortBy) && sortBy.length > 0) {
+    const sorts = sortBy
+      .map((fieldSort) => `${fieldSort[1]}+${fieldSort[0]}`)
+      .join(',');
+    // Direct update on string url to prevent encoding of +A and +D
+    url = `${url}${new URL(url).search ? '&' : ''}SORTBY=${sorts}`;
```

Comment:
@AlitaBernachot the order param is supposed to be "space A" or "space D" and is then encoded as `+A` or `+D` :) so simply using a space in place of the `+` char will let you update the params without this workaround.